torch/csrc/jit/pickler.h

```diff
@@ -11,6 +11,13 @@
 namespace torch {
 namespace jit {
 
+// In order to solve the conflict between the macro and the variable in MSVC,
+// we need to save the value first to `UNICODE_TMP` and undefine `UNICODE`.
+#ifdef UNICODE
+# define UNICODE_TMP UNICODE
+# undef UNICODE
+#endif
+
```

Comment:
@ezyang You mean the one in L41? Could you please suggest a good one?

Reply:
`TORCH_UNICODE`?

Reply:
or maybe `UNICODE_` (trailing underscore being a classic way of sidestepping reserved words)